scripts/rageshake.go

```diff
@@ -110,6 +127,20 @@ func main() {
 	// Make sure bugs directory exists
 	_ = os.Mkdir("bugs", os.ModePerm)
 
+	// serve files under "bugs"
+	fs := http.FileServer(http.Dir("bugs"))
+	fs = http.StripPrefix("/api/listing/", fs)
+
+	// set auth if env vars exist
+	usr := os.Getenv("BUGS_USER")
+	pass := os.Getenv("BUGS_PASS")
+	if usr == "" || pass == "" {
+		fmt.Println("BUGS_USER and BUGS_PASS env vars not found. No authentication is running for /api/listing")
+	} else {
+		fs = basicAuth(fs, usr, pass, "Enter username and password")
+	}
+	http.Handle("/api/listing/", fs)
```

Comment:
We as a team have certain conventions when it comes to Go and HTTP servers. One of them is that everything public is served under `/api` so that you can attach Prometheus metrics (which hook into `/metrics`) and pprof tooling (which hook into `/debug/pprof`) without making those endpoints visible to the wider internet which would be the case if we attached to `/`. See https://docs.google.com/document/d/1yu7qImAcr-_yN72Ndb5ht_zjyK0hxXQGZQYIKFaGBXw/edit